src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -802,18 +804,34 @@ def _create_pgm_input_transformers(self):  # pylint: disable=too-many-statements
         # positive and zero sequence magnetising impedance must be equal.
         # mag0_percent = z0mag / z0.
         checks = {
-            "vk0_percent": np.allclose(vk_percent, vk0_percent) or np.isnan(vk0_percent).all(),
-            "vkr0_percent": np.allclose(vkr_percent, vkr0_percent) or np.isnan(vkr0_percent).all(),
-            "mag0_percent": np.allclose(i_no_load * 1e-2, 1e4 / (vk0_percent * mag0_percent))
-            or np.isnan(mag0_percent).all(),
-            "mag0_rx": np.allclose(rx_mag, mag0_rx) or np.isnan(mag0_rx).all(),
+            "vk0_percent": np.logical_or(np.allclose(vk_percent, vk0_percent), np.isnan(vk0_percent).all()),
+            "vkr0_percent": np.logical_or(np.allclose(vkr_percent, vkr0_percent), np.isnan(vkr0_percent).all()),
             "si0_hv_partial": np.isnan(
                 self._get_pp_attr("trafo", "si0_hv_partial", expected_type="f8", default=np.nan)
             ).all(),
         }
         if not all(checks.values()):
             failed_checks = ", ".join([key for key, value in checks.items() if not value])
             logger.warning("Zero sequence parameters given in trafo shall be ignored: %s", failed_checks)
+        valid = np.logical_and(
```

Comment:
Since `mag0_percent` or `mag0_rx` can be nan, lets also filter them out as invalid.